modules/csg/csg_shape.cpp

```diff
@@ -30,7 +30,317 @@
 
 #include "csg_shape.h"
 
+#include "core/math/aabb.h"
 #include "core/math/geometry_2d.h"
+#include "core/math/math_funcs.h"
```

Comment:
Not all of these are used.

Reply:
All the `core/math` ones are already included, as is `vector` and `ref_counted`